packages/react/test/specs/themes/teams/components/Icon/Icon-test.ts

```diff
@@ -0,0 +1,36 @@
+import * as _ from 'lodash'
+import { shallow } from 'enzyme'
+
+import icons, {
+  teamsIconSlotClassNames,
+} from '../../../../../../src/themes/teams/components/Icon/svg'
+import processedIcons from '../../../../../../src/themes/teams/components/Icon/svg/processedIndex'
+import { SvgIconSpecWithStyles } from '../../../../../../src/themes/teams/components/Icon/svg/types'
+
+describe('Teams Theme Icon', () => {
+  function testIcon(icon) {
+    const iconFunc: any = (icon as SvgIconSpecWithStyles).icon
+    const component = shallow(
+      iconFunc({ classes: { outlinePart: 'TEST-OUTLINE', filledPart: 'TEST-FILLED' } }),
```

Comment:
Actually this a React component, correct?

```suggestion
      <SvgIcon classes={{ outlinePart: 'TEST-OUTLINE', filledPart: 'TEST-FILLED' }} />
```

Can we use JSX there?